### DISCUSSION TITLE
update to new version without losing items I added

### DISCUSSION THREAD

**Discussion by Speedmod88** [jef/streetmerchant#1765] [Help]
Hi
I'm new user of github and probably there a way to do that but I don't know how exactly... so is anyone can tell me how to update the streetmerchant without losing all items I added?

Thank You
Best Regards

**Comment by 85256638** [comment]
I would love to know as well

**Reply by xkuza** [reply]
It shouldn't , as new version doesn't have  "dotenv" config file, only "dotenv-example". 

if in doubt - save /streetmerchant/dotenv config to your safe place, then put it back after update.
